types/errors/errors.go

```diff
@@ -156,6 +156,9 @@ var (
 	// ErrUnsupportedTxType defines an error for an unsupported transaction type (e.g., EIP-4844 transactions)
 	ErrUnsupportedTxType = Register(RootCodespace, 44, "unsupported transaction type")
 
+	// ErrEVMVMError defines an error for an evm vm error (eg. revert)
+	ErrEVMVMError = Register(RootCodespace, 45, "evm vm error")
```

Comment:
What do you think about `evm reverted`?  (Often it's some contract validation that reverts, so it's not really a broken thing)

Reply:
yeah sounds good